geomesa-tools/src/main/assemblies/component.xml

```diff
@@ -24,16 +24,7 @@
             <outputDirectory>bin</outputDirectory>
             <directoryMode>755</directoryMode>
             <fileMode>0755</fileMode>
-            <includes>
-                <include>geomesa</include>
-                <include>install-jai</include>
-                <include>install-jline</include>
-                <include>install-vecmath</include>
-                <include>install-imageio-ext</include>
-                <include>install-hadoop-accumulo.sh</include>
-                <include>install-kafka.sh</include>
-            </includes>
-            <filtered>true</filtered>
+            <filtered>false</filtered>
```

Comment:
don't we need filtered true for e.g. https://github.com/locationtech/geomesa/commit/885d5b60ea5832b1a2a902c3f57d2e0b60e9d8f6#diff-051f96c3d5a44fe0603c870637be7e90R14 ?